index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Alicja Zubel Developer</title>
+	<link rel="icon" href="favicon.ico" type="image/x-icon"/>
+	<link rel="stylesheet" href="normalize.css" type="text/css">
+	<link rel="stylesheet" href="style.css" type="text/css">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap" rel="stylesheet">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet"> 
+</head>
+<body>
+	<div class="container">
+		
+		<section class="menu">
+			<div class="logo">
+				<button class="menu-image">&lt;</button>
+				<a href="index.html"> <img class="menu-logo" src="karate.jpg" alt="logo"> </a>
+			</div>
+			<div class="navigation">
+				<nav>
+					<a class="current" href="#">Home</a>
+					<a href="about.html">About</a>
+					<a href="portfolio.html">Portfolio</a>
+					<a href="contact.html">Contact</a>
+					<a href="">CV</a>
+				</nav>
+			</div>
+			<div class="links">
+				<a  href="https://github.com/AlicjaZub" target="_blank"><img class="menu-image" src="githublight.png" alt="github logo"></a>
+				<a  href="https://www.linkedin.com/in/alicja-zubel-07a9515a/" target="_blank"><img class="menu-image" src="lin.png" alt="linkedin logo"></a>
+			</div>	
+		</section>
+		
+		<section class="right-section">
+			<button class="hidden-menu-button">MENU</button>
+			<div class="info">
+				<div class="left">
+					<h1 class="welcome-heading">Welcome!</h1>
+					<h2 class="welcome-heading2">My name is <span>Alicja</span> and I'm a something something</h2>
```

Comment:
classes with numbers in like`welcome-heading2` can often be improved. this isn't the second heading, its the sub-heading. which makes a better class name